Dockerfile

```diff
@@ -192,16 +197,33 @@ RUN --mount=target=/root/.cache,type=cache \
   xx-verify --static /out/containerd-stargz-grpc && \
   xx-verify --static /out/ctr-remote
 
-FROM buildkit-export AS buildkit-linux
+# Copy together all binaries needed for oci worker mode
+FROM buildkit-export AS buildkit-buildkitd.oci_only
+COPY --link --from=buildkitd.oci_only /usr/bin/buildkitd.oci_only /usr/bin/
+COPY --link --from=buildctl /usr/bin/buildctl /usr/bin/
+ENTRYPOINT ["buildkitd.oci_only"]
+
+# Copy together all binaries for containerd worker mode
+FROM buildkit-export AS buildkit-buildkitd.containerd_only
+COPY --link --from=buildkitd.containerd_only /usr/bin/buildkitd.containerd_only /usr/bin/
+COPY --link --from=buildctl /usr/bin/buildctl /usr/bin/
+ENTRYPOINT ["buildkitd.containerd_only"]
```

Comment:
Not sure these parts are relevant to FreeBSD

Reply:
Perhaps a bad rebase with https://github.com/moby/buildkit/commit/7ba07b60b07c55a5a0bd68fdb72ceab8182409c6

Reply:
Oh, indeed. Dockerfile needed more attention than I thought. Thanks!